gax/src/main/java/com/google/api/gax/core/GoogleCredentialsProvider.java

```diff
@@ -49,10 +49,18 @@
 @AutoValue
 public abstract class GoogleCredentialsProvider implements CredentialsProvider {
 
-  public abstract List<String> getScopesToApply();
+  public List<String> getScopesToApply() {
+    return getScopesToApplyInner();
+  }
+
+  abstract ImmutableList<String> getScopesToApplyInner();
 
   @BetaApi
-  public abstract List<String> getJwtEnabledScopes();
+  public List<String> getJwtEnabledScopes() {
+    return getJwtEnabledScopesInner();
+  }
+
+  abstract ImmutableList<String> getJwtEnabledScopesInner();
```

Comment:
All the other changes (like this one with `*Inner` methods) are not needed anymore. Please rollback them.